install.sh

```diff
@@ -59,19 +57,21 @@ cleanup() {
 }
 
 trap cleanup EXIT
+
+echo Downloading the archive...
```

Comment:
This extra log line can be removed, curl is configured below to emit a progress bar, so it'll be clear that something is going on

Reply:
I've added this when I saw some progress bar, but didn't see what it is about. It's a bit scary to see something happen on your laptop, and it's continuous process, but you don't know what it does.

Reply:
Ah, okay got it. Can we say "Downloading CodeCrafters CLI..." or something like that ("archive" isn't important from the user's perspective).